rewrite-java-test/src/test/java/org/openrewrite/java/OrderImportsTest.java

```diff
@@ -686,4 +684,78 @@ void foldPackageWithExistingImports() {
           )
         );
     }
+
+    @Test
+    void nestedInnerClass() {
+        // language=java
+        rewriteRun(
+          spec -> spec.recipe(new OrderImports(true)),
```

Comment:
I only see code changes in `ImportLayoutStyle` and `OrderImports`. Is it necessary to test through `new OrderImports(true)` here, or can we move these tests to `OrderImportsTest` ?

Reply:
I think we can move it!